### PR TITLE
fix for more recent Ubuntu and Debian

### PR DESCRIPTION
**Pull request by AndreasPfaffeneder**

### CODE REVIEW

**Comment by igalic** [comment]
these run failures seem to be perfectly unrelated to your changes…

File: manifests/params.pp [hunk 1]
```diff
@@ -36,14 +36,19 @@
...
         $ruby_pkg = 'ruby-augeas'
       }
-      elsif versioncmp($::rubyversion, '1.9.1') >= 0 {
-        $ruby_pkg = 'libaugeas-ruby1.9.1'
-      } else {
-        $ruby_pkg = 'libaugeas-ruby1.8'
-      }
+      else {
+        if versioncmp($::rubyversion, '2.3.0') == 0 {
+          $ruby_pkg = 'ruby-augeas'
```
Comment (igalic) [inline manifests/params.pp]: i'm confused… on both, Debian and Ubuntu the ruby version is >= 2.10 (2.3.0 ***is*** >= 2.1.0), and on *both* the package name is 'ruby'augeas'

so why do we need a special case for 2.3.0?

Comment (raphink) [inline manifests/params.pp]: Yes, I don't really understand how that changes anything…

**Comment by AndreasPfaffeneder** [comment]
.

**Pull request closed without merging**